### PR TITLE
Feature/boolean

### PR DESCRIPTION
**Pull request by agistyaanugrah**
Add Feature boolean

### CODE REVIEW

File: tests/test_boolean.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+import unittest
+
+import random
+import pandas as pd
+
+from datasae.boolean import Boolean, WarningDataDetailMessage
+from datasae.utils import create_warning_data, WarningDataMessage
+
+from . import MESSAGE
```
Comment (pipinfitriadi) [inline tests/test_boolean.py:14]: @agistyaanugrah 
```py
import random
import unittest

import pandas as pd

from . import MESSAGE
from datasae.boolean import Boolean, WarningDataDetailMessage
from datasae.utils import create_warning_data, WarningDataMessage
```

**Review by pipinfitriadi — CHANGES_REQUESTED** [review]

File: tests/test_boolean.py (hunk 1 above)
Comment (pipinfitriadi) [reply tests/test_boolean.py:14]: Coverage Unittesting nya belum 100% @agistyaanugrah 

https://github.com/jabardigitalservice/DataSae/actions/runs/6323317661/job/17170807808?pr=8

Name                      Stmts   Miss  Cover   Missing
-------------------------------------------------------
datasae/__init__.py           0      0   100%
datasae/boolean.py           67      6    91%   52-53, 89, 131-132, 170
datasae/exception.py         28      0   100%
datasae/float.py            240      0   100%
datasae/integer.py          269      0   100%
datasae/utils.py             11      0   100%
tests/__init__.py             1      0   100%
tests/test_boolean.py        30      0   100%
tests/test_exception.py       7      0   100%
tests/test_float.py          71      0   100%
tests/test_integer.py       100      0   100%
-------------------------------------------------------
TOTAL                       824      6    99%

**Review by pipinfitriadi — CHANGES_REQUESTED** [review]
mohon bantuan fixingnya kakak @agistyaanugrah

File: tests/test_boolean.py (hunk 1 above)
Comment (agistyaanugrah) [reply tests/test_boolean.py]: yang coverage pasti gaakan 100% dari functionya karna pasti ke cover sama function isinstance

**Review by pipinfitriadi — CHANGES_REQUESTED** [review]
sip tinggal coverage test nya ya @agistyaanugrah

**Comment by agistyaanugrah** [comment]
initu kalau code nya kayak gini gaakan bisa 100 coverage nya karna udah ke filter di awal pake yg isintace boolean @adibintangprada

**Comment by adibintangprada** [comment]
> initu kalau code nya kayak gini gaakan bisa 100 coverage nya karna udah ke filter di awal pake yg isintace boolean @adibintangprada

Berarti yg di staticmethod gaperlu pake isinstance? isinstancenya di yg basic function aja ya?

**Review by pipinfitriadi — APPROVED** [review]
well done @agistyaanugrah @adibintangprada

**Pull request merged**